utils/config_generator.py

```diff
@@ -0,0 +1,383 @@
+        group_corpus_name = normalize_corpus_name(entry.did.group + entry.did.name)
+        if not (corpus_name in visited_corpora or group_corpus_name in visited_corpora):
+            train_datasets.append(corpus_key)
+            byte_size, display_size = get_remote_file_size(entry.url)
+            if byte_size:
+                # Don't add the sentences to the total, as these will be commented out by default.
+                sentences = estimate_sentence_size(byte_size)
+                train_datasets.yaml_add_eol_comment(
+                    f"~{sentences:,} sentences ".rjust(70 - len(corpus_key), " ")
+                    + f"({display_size})",
+                    len(train_datasets) - 1,
+                )
+
+    train_comment = "\n".join(
+        [
+            "The training data contains:",
+            f"  {total_sentences:,} sentences",
+            "",
+            "Skipped datasets:",
+            *[f" - {d}" for d in skipped_datasets],
+        ]
+    )
+
+    comment_section["  train:"] = train_comment
+
+
+def normalize_corpus_name(corpus_name: str):
+    """Normalize the corpus name so that it's easy to deduplicate between opus and mtdata."""
+
+    # Remove the language tags at the end.
+    # mtdata_ELRC-vnk.fi-1-eng-fin
+    #                     ^^^^^^^^
+    corpus_name = re.sub(r"-\w{3}-\w{3}$", "", corpus_name)
+
+    corpus_name = corpus_name.lower()
+
+    # Remove numbers anything that is not a letter. This is a little aggressive, but should help
+    # deduplicate more datasets. For example:
+    #   opus: 725-Hallituskausi_2011_2
+    #   mtdata: hallituskausi_2011_2015-1-eng-fin
+    corpus_name = re.sub(r"[^a-z]", "", corpus_name.lower())
+
+    # Ignore train/test/dev splits.
+    #   opus: NeuLab-TedTalks/v1
+    #   mtdata: Neulab-tedtalks_train-1-eng-fin
+    #   mtdata: Neulab-tedtalks_test-1-eng-fin
+    #   mtdata: Neulab-tedtalks_dev-1-eng-fin
+    corpus_name = re.sub(r"train$", "", corpus_name)
+    corpus_name = re.sub(r"test$", "", corpus_name)
+    corpus_name = re.sub(r"dev$", "", corpus_name)
+
+    return corpus_name
+
+
+def add_test_data(
+    source: str,
+    target: str,
+    test_datasets: list[str],
+    devtest_datasets: list[str],
+    comment_section: dict[str, str],
+):
+    skipped_datasets = []
+    print("Fetching flores")
+    if source in flores_101_languages and target in flores_101_languages:
+        test_datasets.append("flores_devtest")
+
+        # Add augmented datasets to check performance for the specific cases
+        test_datasets.append("flores_aug-mix_devtest")
+        test_datasets.append("flores_aug-title_devtest")
+        test_datasets.append("flores_aug-upper_devtest")
+        test_datasets.append("flores_aug-typos_devtest")
+        test_datasets.append("flores_aug-noise_devtest")
+        test_datasets.append("flores_aug-inline-noise_devtest")
+
+        devtest_datasets.append("flores_aug-mix_dev")
```

Comment:
All devtest datasets should be with `aug-mix` augmentation

Reply:
As far as I can tell they all do have `aug-mix` specified here.